src/main/java/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -350,4 +355,15 @@ private Study phaseTransition(String appId, String studyId, StudyPhase targetPha
         
         return study;
     }
+    
+    private Set<String> getCustomEventIdsFromSchedule(String appId, String scheduleGuid) {
+        Schedule2 schedule = scheduleService.getSchedule(appId, scheduleGuid);
```

Comment:
A study can not have a scheduleGuid, so you can't guarantee it won't be null here.

Reply:
Added null check on scheduleGuid before calling getSchedule.